quarkchain/cluster/jsonrpc.py

```
@@ -1431,7 +1431,7 @@ def shutdown(self):
         await websocket.send(json.dumps(response))
 
     @public_methods.add
-    async def subscribe(self, sub_type, full_shard_id, context):
+    async def subscribe(self, sub_type, full_shard_id, params=None, context=None):
         if context is None or full_shard_id is None:
             return None
```

maybe `raise ValueError("Unexpected subscription request")`?